lago/utils.py

```diff
@@ -690,6 +690,65 @@ def get_hash(file_path, checksum='sha1'):
     return sha.hexdigest()
 
 
+def filter_spec(spec, paths):
+    """
+    Remove keys from a spec file.
+    For example, with the following path: domains/*/disks/*/metadata
+    all the metadata dicts from all domains disks will be removed.
+
+    Args:
+        spec (dict): spec file to remove keys from
+        paths (list): list of paths to the keys that should be removed
+
+    Returns:
+        None
+    """
+
+    def remove_key(path, spec):
+        if len(path) == 0:
+            return
+        elif len(path) == 1:
+            key = path.pop()
+            if not isinstance(spec, dict):
+                raise LagoUserException(
```

Comment:
I would print a more detailed message here, that hints what to do or what the user did wrong.